lib/rubocop/cop/lint/enable_statement.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# rubocop:disable Lint/UnneededDisable
+module RuboCop
+  module Cop
+    module Lint
+      # @example
+      #   # good
+      #   # rubocop:disable Layout/SpaceAroundOperators
+      #   # x= 0
```

Comment:
The code parts of the example shouldn't have a leading `#`, because they will be shown as comments in the documentation,